doc/xsd/tester_defs.xsd

```diff
@@ -334,6 +334,31 @@
         </attribute>
     </attributeGroup>
 
+    <complexType name="Run-Template">
+        <annotation>
+            <documentation>
+                Default argument Template.
+            </documentation>
+        </annotation>
+        <sequence>
+            <element name="script" type="Script" minOccurs="0">
+                <annotation>
+                    <documentation>
+                        Default script for template.
+                    </documentation>
+                </annotation>
+            </element>
+            <group ref="Arguments"/>
+        </sequence>
+        <attribute name="name" type="Name">
```

Comment:
Please, carefully analyze which run-item attributes are applicable to run templates

Reply:
Could I add only script name and arguments for now in this patch? And further additions for run-template will be added in future if it will be needed.